### PR TITLE
UI: Play and pause glyph added

### PR DESCRIPTION
**Pull request by backermann1978**
Added glyph icons for play and pause at dev conf in Cologne.

### CODE REVIEW

**Review by Amstutz — CHANGES_REQUESTED** [review]
Hi @backermann1978 

Thank you a lot for contributing to the UI Framework of ILIAS.

It's nice to see such a well worked out PR sparking from the workshop at the last Dev-Conf.

Please answer the following questions:
- [ ] Use Case: Do you see a concrete use case in ILIAS, where this glyph will be used? Is there some use case in an upcoming feature?
- [ ] Inside Video Frame: Can you be a bit more specific on what "Inside a video frame means?"
- [ ] Replacement of Glyphs: Since you will not want to reload the complete page if one clicks pause, you will have to do that with JS. Is the plan here, to attach manually some JS handling the player and the glyph?

Please implement the following changes:
- [ ] Effect Play: Move the description: "The Play Glyph should be replaced by Pause Glyph while the video is played." to Effect. Same for pause.

Note that this is just the review for the interface. If the JF accepts the interface we will provide an other review on the implementation before merging.

kindly,
@Amstutz

File: src/UI/Component/Glyph/Factory.php [hunk 1]
```diff
@@ -834,4 +834,58 @@ public function reset($action = null);
...
+     *   effect: >
+     *      Clicking the Pause Glyph pauses the video presentation.
+     *
+     *
+     * rules:
+     *   usage:
+     *       1: >
+     *          The Pause Glyph MUST indicate if the video Container Collection is playing a video.
+     *       2: >
+     *          The Play Glyph MUST NOT be shown when the video is playable, but not already playing.
```
Comment (matthiaskunkel) [inline src/UI/Component/Glyph/Factory.php:881]: Please change to: The Pause glyph MUST NOT be shown when the video is already playing.

**Comment by matthiaskunkel** [comment]
Jour Fixe, 08 APR 2019 : Pull request is highly appreciated and accepted for trunk. Please include Timon's requests and change the rule according to Matthias' comment.

**Comment by Amstutz** [comment]
Let me know if any help assistance is needed with this PR.

**Comment by Amstutz** [comment]
Any update?

**Comment by klees** [comment]
As @backermann1978 does not seem to work on this, I would propose to close this.

Maybe someone (@nhaagen, @Amstutz, me) could duplicate the branch to our own repositories so we won't lose the code even if @backermann1978 should disappear =)

**Comment by backermann1978** [comment]
Hi @Amstutz and @klees, sorry that I couldn't work on my PR in the last weeks and didn't answer to your messages. I would like to resume work if this is still ok?! And I will try to not disappear... ;-)

**Comment by Amstutz** [comment]
Since this seems inactive for a while, I close this down. Please re-open as soon as this can be re-evaluated. thx a lot for your contribution.

**Pull request closed without merging**